src/protocol/protocol.yml

```diff
@@ -1798,6 +1798,13 @@ Response:
     url: string
     status: number
     statusText: string
+    requestHeaders:
```

Comment:
Not: `rawheaders` might be a more descriptive name

Reply:
These are request headers on response object!